src/assets/templates/typescript/tslint.json

```diff
@@ -4,6 +4,22 @@
       "tslint-config-airbnb"
   ],
   "jsRules": {},
-  "rules": {},
+  "rules": {
+    "max-line-length": [
+      false
+    ],
+    "triple-equals":[
```

Comment:
✅ This makes good sense because we can almost never guarantee value types in comparisons.